aws/modules/deploy/helms/gpii-preferences/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for gpii-preferences.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+svc_listen_port: 80
+preferences_listen_port: 8081
+node_env: gpii.config.preferencesServer.standalone.production
+
+image:
+  repository: gpii/universal
+  checksum: sha256:57d0a17da505c8bc28c221c88b8de54447c78dc873c355d0bfd435d28f8b09ee
+  pullPolicy: IfNotPresent
+
+issuerRef:
+  name: letsencrypt-production
+
+dnsNames:
+- preferences.stg.gpii.net
```

Comment:
This hardcoded value looks wrong.

Reply:
What default value should go here instead?